src/pages/projects.js

```diff
@@ -378,11 +378,12 @@ const ProjectDescriptions = () => (
         </div>
         <div className="project-description">
           <p>
-            Major:
+            Major:{" "}
```

Comment:
How about setting `Major:` and `Minor:` in bold? I thought it helped dissociating the list from the descriptions below (see comparison in #160).
```suggestion
            <b>Major:</b>{" "}
```

Reply:
Done